bundles/org.openhab.binding.freeathomesystem/src/main/feature/feature.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
This file doesn't need a license header.

Reply:
Fix is done